openedx/core/lib/block_structure/tests/test_block_structure.py

```diff
@@ -131,6 +131,7 @@ def test_xblock_data(self):
         block_structure = BlockStructureModulestoreData(root_block_usage_key=0)
         for block in blocks:
             block_structure._add_xblock(block.location, block)
+            block_structure._get_or_create_block(block.location)
```

Comment:
We should spend some more time looking at the ergonomics of the block structure interface, and see if we can make it harder to do the wrong thing. (Not in this PR, though)